integration/benchnet2/automate/templates/helm-values-all-nodes.yml

```diff
@@ -0,0 +1,385 @@
+          value: "true"
+        - name: OTEL_RESOURCE_ATTRIBUTES
+          value: network
+        - name: BINSTAT_ENABLE
+          value: ""
+        - name: BINSTAT_LEN_WHAT
+          value: ""
+        - name: BINSTAT_DMP_NAME
+          value: ""
+        - name: BINSTAT_DMP_PATH
+          value: ""
+      image: gcr.io/flow-container-registry/access:v0.27.6
+      nodeId: {{$val.node_id}}
+{{end}}{{end}}
+collection:
+  defaults:
+    imagePullPolicy: Always
+    containerPorts:
+      - name: ptp
+        containerPort: 3569
+      - name: grpc
+        containerPort: 9000
+      - name: secure-grpc
+        containerPort: 9001
+      - name: admin
+        containerPort: 9002
+    env: []
+    resources:
+      requests:
+        cpu: "200m"
+        memory: "128Mi"
+      limits:
+        cpu: "800m"
+        memory: "10Gi"
+    servicePorts:
+      - name: ptp
+        protocol: TCP
+        port: 3569
+        targetPort: ptp
+      - name: grpc
+        protocol: TCP
+        port: 9000
+        targetPort: grpc
+      - name: secure-grpc
+        protocol: TCP
+        port: 9001
+        targetPort: secure-grpc
+      - name: admin
+        protocol: TCP
+        port: 9002
+        targetPort: admin
+    storage: 1G
+  nodes:
+{{range $val := .}}{{if eq ($val.role) ("collection")}}
+    {{$val.name}}:
+      args:
+        - --bootstrapdir=/bootstrap
+        - --datadir=/data/protocol
+        - --secretsdir=/data/secret
+        - --loglevel=INFO
+        - --profiler-enabled=false
+        - --profile-uploader-enabled=false
+        - --tracer-enabled=false
+        - --bind=0.0.0.0:3569
+        - --profiler-dir=/profiler
+        - --profiler-interval=2m
+        - --nodeid={{$val.node_id}}
+        - --block-rate-delay=950ms
+        - --hotstuff-timeout=2.15s
+        - --hotstuff-min-timeout=2.15s
+        - --ingress-addr=0.0.0.0:9000
+        - --insecure-access-api=false
+        - --access-node-ids=*
+      env:
+        - name: OTEL_EXPORTER_OTLP_TRACES_ENDPOINT
+          value: http://tempo:4317
+        - name: OTEL_EXPORTER_OTLP_TRACES_INSECURE
+          value: "true"
+        - name: OTEL_RESOURCE_ATTRIBUTES
+          value: network
+        - name: BINSTAT_ENABLE
+          value: ""
+        - name: BINSTAT_LEN_WHAT
+          value: ""
+        - name: BINSTAT_DMP_NAME
+          value: ""
+        - name: BINSTAT_DMP_PATH
+          value: ""
+      image: gcr.io/flow-container-registry/collection:v0.27.6
+      nodeId: {{$val.node_id}}
+{{end}}{{end}}
+consensus:
+  defaults:
+    imagePullPolicy: Always
+    containerPorts:
+      - name: ptp
+        containerPort: 3569
+      - name: grpc
+        containerPort: 9000
+      - name: secure-grpc
+        containerPort: 9001
+      - name: admin
+        containerPort: 9002
+    env: []
+    resources:
+      requests:
+        cpu: "200m"
+        memory: "128Mi"
+      limits:
+        cpu: "800m"
+        memory: "10Gi"
```

Comment:
these limits are not suitable for benchmarking.

Reply:
For now, I would rather have a template with "scaled down" resources so that we can use this template for testing. I don't want to unintentionally be spinning up containers that will consume a lot of resources.

These can definitely be updated later to align with a network that will be used for benchmarking, or we can create a whole new template that will specifically be used for this purpose.